website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -141,6 +145,26 @@ To delete a policy set:
 
 The policy set no longer appears on the UI and Terraform Cloud no longer applies it to any workspaces. For managed policy sets, all of the individual policies are still available in Terraform Cloud. You must delete each policy individually to remove it from your organization.
 
+### Pinning policy tool versions to policy sets
```

Comment:
@mrinalirao Having reviewed the page outline, I think it may be advisable to place this into it's own section. What do you think?